### ISSUE TITLE
Zoom while dragging

### ISSUE THREAD

**Issue by TDHster** [stuffmatic/fSpy#7]
Need a zoomed area while dragging point for preciese postioning. (Like in blender while placing motiontracking points). It will be allow pixel precision without any additional sortcuts or user action.
This area will be good in left or right panel, or near cursor while drag and drop point.

**Comment by mattwolcott** [comment]
This is already implemented, press Shift while dragging

**Comment by TDHster** [comment]
Yes, but shift isn't evident for new users and side panels have enough space for this.

**Comment by walles** [comment]
Related to #22
